### ISSUE TITLE
[Breaking change]: System.Windows.Forms.StatusStrip is using a different default renderer

### ISSUE THREAD

**Issue by Tanya-Solyanik** [dotnet/docs#43957]
### Description

Related to https://github.com/dotnet/winforms/issues/11450  - focus indicator over the StatusBar ToolStripSplitButton control is hard to see because color contrast ratio is too low.

### Version

.NET 9

### Previous behavior

[System.Windows.Forms.StatusBar](https://learn.microsoft.com/en-us/dotnet/desktop/winforms/controls/statusstrip-control-overview?view=netframeworkdesktop-4.8&viewFallbackFrom=netdesktop-9.0) control was using ToolStripRenderMode.System as the default value for [RenderMode](https://learn.microsoft.com/en-us/dotnet/api/system.windows.forms.toolstrip.rendermode?view=windowsdesktop-9.0#system-windows-forms-toolstrip-rendermode) property. However the corresponding renderer was creating inaccessible UI, the focus indicator over the  StatusBar's ToolStripSplitButton control was hard to see because the color contrast ratio was too low. 

### New behavior

We now use the default renderer to draw controls in the StatusStrip. As a result, controls look different in .NET 9. The down arrow in ToolStripSplitButtons in a little smaller, and if the parent control background is non-default, then the user will see a line above the StatusStrip.

### Type of breaking change

- [ ] **Binary incompatible**: Existing binaries might encounter a breaking change in behavior, such as failure to load or execute, and if so, require recompilation.
- [ ] **Source incompatible**: When recompiled using the new SDK or component or to target the new runtime, existing source code might require source changes to compile successfully.
- [x] **Behavioral change**: Existing binaries might behave differently at run time.

### Reason for change

We are now using a more accessible default per Accessibility requirements. Related to https://github.com/dotnet/winforms/issues/11450 

### Recommended action

We recommend using the new default value to satisfy the color-contrast requirements. If the layout compatibility is required, then set StatusStrip.RenderMode to ToolStripRenderMode.System.

### Feature area

Windows Forms

### Affected APIs

[StatusStrip.RenderMode](https://learn.microsoft.com/dotnet/api/system.windows.forms.toolstripmanager.rendermode?view=windowsdesktop-9.0)

**Comment by Tanya-Solyanik** [comment]
@CamSoper  - we are redoing the fix that caused this issue as we discovered that the fix was incomplete. The new fix does not have the rendering artifacts, and we are servicing the new fix to NET9. We would need to revert the breaking change update. Should I open a new breaking issue?

**Comment by CamSoper** [comment]
@Tanya-Solyanik Yes, please. It helps our tracking. :) Thanks!
